proto/decentraland/sdk/components/light_source.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package decentraland.sdk.components;
+import "decentraland/sdk/components/common/id.proto";
+import "decentraland/common/colors.proto";
+option (common.ecs_component_id) = 1079;
+
+message PBLightSource {
+  bool active = 4;                          // default = true
+  decentraland.common.Color3 color = 1;     // default = Color.white
+  float brightness = 2;                     // range from 1 (dim) to 100,000 (very bright), expressed in Lumens for Point and Spot
+  float range = 3;                          // default = 10, expressed in meters
+  ShadowType shadow = 5;                    // default = ShadowType.ST_NONE
```

Comment:
We believe that the default value of `range` should be based on `brightness`.

Additionally, if you want to implement this with a "default" value, you should mark it as `optional float range = 3`.

Only "optional" values can have default values, which are not required to be specified in the SDK (you will receive `undefined` or `null` in Unity, and you need to interpret that as the default).